app/Http/Controllers/Api/V1/Integration/VideoDirectUrlController.php

```diff
@@ -82,8 +83,11 @@ private function getVimeoVideoInfo($url)
     }
 
     /**
+     * Get video config
```

Comment:
Add new line between title, description param and return.

Reply:
Done!